.pylintrc-tests

```diff
@@ -115,3 +119,14 @@ disable=
 	# if:/else: and for:/else:.
 	useless-else-on-loop,
 	no-else-return,
+	# NOTE(lidiz): Python 3 make object inheritance default, but not PY2
+	useless-object-inheritance,
+	# NOTE(lidiz): Limit the length of function doesn't mean reducing
```

Comment:
As with the `.pylintrc`, I don't see sufficient reason to disable any of these (other than `useless-object-inheritance`) globally